03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+def calculate_strike_points(next_frame, after_next_frame)
+  if next_frame[0] == 10 && after_next_frame[0] == 10
+    30
+  elsif next_frame[0] == 10
+    20 + after_next_frame[0]
```

Comment:
```suggestion
  if next_frame[0] == 10
    after_next_frame[0] == 10 ? 30 : 20 + after_next_frame[0]
```

`if next_frame[0] == 10` が2回出てくるのでこんなふうに書くと1回にできます。
が、元のコードもシンプルでわかりやすいので、無理に直さなくてもいいかも